StudCity.Frontend/src/components/Group/GroupCard.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import GridCard from './GridCard'
 import ListCard from './ListCard'
 
-const GroupCard = ({ group, handleClick, viewMode }) => {
+const GroupCard = ({ group, handleClick, viewMode = 'grid' }) => {
```

Comment:
What about doing like this?

```
export const ViewMode = {
      grid: 'grid',
...
}
```

Reply:
@maryanchuk04 why though?👀

Reply:
I don`t understand what you are talking about